pkg/config/appsec.go

```diff
@@ -0,0 +1,9 @@
+package config
+
+// setupAppSec initializes the configuration values of the appsec agent.
+func setupAppSec(cfg Config) {
+	cfg.SetKnown("appsec_config.enabled")
+	cfg.SetKnown("appsec_config.appsec_dd_url")
```

Comment:
The two options below automatically set things as "known", so you can remove these two lines.

Reply:
Fixed 👍